### PR TITLE
Fix agglayer E2E test

### PR DESCRIPTION
**Pull request by Stefan-Ethernal**
The PR fixes failing E2E test for the agglayer, by updating the CDK validium node Docker image and config.

### CODE REVIEW

**Review by vcastellm — APPROVED** [review]
LGTM!

File: docker/data/zkevm/node.toml [hunk 1]
```diff
@@ -32,8 +32,8 @@ Outputs = ["stderr"]
 FreeClaimGasLimit = 1500000
 IntervalToRefreshBlockedAddresses = "5m"
 IntervalToRefreshGasPrices = "5s"
-MaxTxBytesSize = 100132
-MaxTxDataBytesSize = 100000
+MaxTxBytesSize=100132
+MaxTxDataBytesSize=100000
```
Comment (Stefan-Ethernal) [inline docker/data/zkevm/node.toml:36]: Nitpick, to have consistent formatting
```suggestion
MaxTxBytesSize = 100132
MaxTxDataBytesSize = 100000
```

File: docker/docker-compose.yaml [hunk 2]
```diff
@@ -137,7 +137,6 @@ services:
         condition: service_started
     volumes:
       - ./data/zkevm/sequencer.keystore:/pk/sequencer.keystore
-      - ./data/zkevm/aggregator.keystore:/pk/aggregator.keystore
```
Comment (Stefan-Ethernal) [inline docker/docker-compose.yaml:140]: Should we revert this perhaps? From what I can see, we are starting the aggregator component (https://github.com/0xPolygon/agglayer/pull/90/files#diff-89fafc265c1fa601cf2364b20be83308031335fda9467fcf3249d5ec1c0c8172R123). There might be some error logging in the node logs, but it is not a critical one I'd say.

File: docker/data/zkevm/node.toml [hunk 3]
```diff
@@ -148,15 +146,15 @@ CleanupLockedProofsInterval = "2m"
...
+AggLayerTxTimeout = "10s"
 AggLayerURL = "http://agglayer:4444"
 SequencerPrivateKey = { Path = "/pk/sequencer.keystore", Password = "testonly" }
+BatchProofL1BlockConfirmations = 2
 
 [EthTxManager]
 ForcedGas = 0
 PrivateKeys = [
 	{Path = "/pk/sequencer.keystore", Password = "testonly"},
-	{Path = "/pk/aggregator.keystore", Password = "testonly"}
```
Comment (Stefan-Ethernal) [inline docker/data/zkevm/node.toml:159]: Should we revert this maybe?

File: docker/data/zkevm/node.toml [hunk 4]
```diff
@@ -167,7 +165,7 @@ DefaultGasPriceWei = 1000000000
 MaxGasPriceWei = 0
 
 [MTClient]
-URI = "zkevm-prover:50061"
+URI  = "zkevm-prover:50061"
```
Comment (Stefan-Ethernal) [inline docker/data/zkevm/node.toml:168]: Formatting
```suggestion
URI = "zkevm-prover:50061"
```

File: docker/data/zkevm/node.toml (hunk 1 above)
Comment (goran-ethernal) [reply docker/data/zkevm/node.toml]: Comments: https://github.com/0xPolygon/agglayer/pull/90/commits/986390a1c514a7a2bdc6730c34a9c9e78f3d9bd1

File: docker/data/zkevm/node.toml (hunk 3 above)
Comment (goran-ethernal) [reply docker/data/zkevm/node.toml]: Fix: https://github.com/0xPolygon/agglayer/pull/90/commits/986390a1c514a7a2bdc6730c34a9c9e78f3d9bd1

File: docker/data/zkevm/node.toml (hunk 4 above)
Comment (goran-ethernal) [reply docker/data/zkevm/node.toml]: Fix: https://github.com/0xPolygon/agglayer/pull/90/commits/986390a1c514a7a2bdc6730c34a9c9e78f3d9bd1

**Pull request merged**